test/e2e-ansible/e2e_ansible_cluster_test.go

```diff
@@ -29,54 +29,42 @@ import (
 )
 
 var _ = Describe("Running ansible projects", func() {
-	var controllerPodName string
-	var memcachedSampleFile string
-	var fooSampleFile string
-	var memfinSampleFile string
-	var memcachedDeployment string
-	var metricsClusterRoleBindingName string
+
+	var (
+		controllerPodName, memcachedDeploymentName, metricsClusterRoleBindingName string
+		fooSampleFile, memfinSampleFile, memcachedSampleFile                      string
+	)
 
 	Context("built with operator-sdk", func() {
 		BeforeEach(func() {
 			metricsClusterRoleBindingName = fmt.Sprintf("%s-metrics-reader", tc.ProjectName)
-
-			By("checking samples")
-			memcachedSampleFile = filepath.Join(tc.Dir, "config", "samples",
+			samplesDir := filepath.Join(tc.Dir, "config", "samples")
+			fooSampleFile = filepath.Join(samplesDir, fmt.Sprintf("%s_%s_foo.yaml", tc.Group, tc.Version))
+			memfinSampleFile = filepath.Join(samplesDir, fmt.Sprintf("%s_%s_memfin.yaml", tc.Group, tc.Version))
+			memcachedSampleFile = filepath.Join(samplesDir,
 				fmt.Sprintf("%s_%s_%s.yaml", tc.Group, tc.Version, strings.ToLower(tc.Kind)))
-			fooSampleFile = filepath.Join(tc.Dir, "config", "samples",
-				fmt.Sprintf("%s_%s_foo.yaml", tc.Group, tc.Version))
-			memfinSampleFile = filepath.Join(tc.Dir, "config", "samples",
-				fmt.Sprintf("%s_%s_memfin.yaml", tc.Group, tc.Version))
 
 			By("deploying project on the cluster")
-			err := tc.Make("deploy", "IMG="+tc.ImageName)
-			Expect(err).NotTo(HaveOccurred())
+			Expect(tc.Make("deploy", "IMG="+tc.ImageName)).To(Succeed())
 		})
+
 		AfterEach(func() {
-			By("deleting Curl Pod created")
-			_, _ = tc.Kubectl.Delete(false, "pod", "curl")
+			By("deleting curl pod")
+			testutils.WrapWarnOutput(tc.Kubectl.Delete(false, "pod", "curl"))
 
-			By("deleting CR instances created")
-			_, _ = tc.Kubectl.Delete(false, "-f", memcachedSampleFile)
-			_, _ = tc.Kubectl.Delete(false, "-f", fooSampleFile)
-			_, _ = tc.Kubectl.Delete(false, "-f", memfinSampleFile)
+			By("deleting test CR instances")
+			for _, sample := range []string{memcachedSampleFile, fooSampleFile, memfinSampleFile} {
+				testutils.WrapWarnOutput(tc.Kubectl.Delete(false, "-f", sample))
+			}
 
 			By("cleaning up permissions")
-			_, _ = tc.Kubectl.Command("delete", "clusterrolebinding",
-				metricsClusterRoleBindingName)
```

Comment:
the reason. for the break in the lines is for not be required to scroll in the IDE to check the code.

Reply:
I think these lines I changed should still be < 120 chars wide.